hrim.py

```diff
@@ -0,0 +1,98 @@
+import sys
+sys.path.insert(0, 'scripts')
+import printing
+import parsing
+import compiling
+import os
+from classes import *
+
+def main(args):
+	if args[1] == "show":
```

Comment:
add `help` avoiding error when execute it

Reply:
Current functionality (implemented in https://github.com/erlerobot/HRIM/pull/16/commits/dcf32ad13ae49e7c7fd3882e4426d82849127650) shows script usage when using the `-h` or `--help` parameters, or when not specifying any parameter (i.e. `python hrim.py`)